ui/components/DataTable.tsx

```diff
@@ -616,6 +616,9 @@ export const DataTable = styled(UnstyledDataTable)`
     overflow: hidden;
     text-overflow: ellipsis;
   }
+  .filter-options-chip {
```

Comment:
💡 Adding the class name on DataTable will change the color of all the chips - at least in oss. Do we need the class name to have the og color in enterprise somewhere else or can we just change it inside ChipGroup?